src-docs/src/views/loading/playground.js

```diff
@@ -66,29 +39,6 @@ export const loadingChartConfig = () => {
   };
 };
 
-export const loadingLogoConfig = () => {
-  const docgenInfo = Array.isArray(OuiLoadingLogo.__docgenInfo)
-    ? OuiLoadingLogo.__docgenInfo[0]
-    : OuiLoadingLogo.__docgenInfo;
-  const propsToUse = propUtilityForPlayground(docgenInfo.props);
-  propsToUse.logo = iconValidator(propsToUse.logo);
-
-  return {
-    config: {
-      componentName: 'OuiLoadingLogo',
-      props: propsToUse,
-      scope: {
-        OuiLoadingLogo,
-      },
-      imports: {
-        '@opensearch-project/oui': {
-          named: ['OuiLoadingLogo'],
-        },
-      },
-    },
-  };
-};
```

Comment:
Hello @ashwin-pc, @BSFishy. 
1. Are we going to show OuiLoadingLogo component in OUI Docs? 
2. As for me, the name of this component does not correctly describe how it can be used. We can use any type that supports OuiIcon, i.e. this does not only apply to Logo, and in this way we can animate any icon. What do you think about it? Should we rename it?

Reply:
Good callout, @KrooshalUX what do you think?

Reply:
While this is technically true that any icon can be swapped - the purpose of this is to provide a branded loading component. Calling it'OuiLoadingLogo' for easy reference , which is what this specific loader is meant to do (animate the brand logo). 
 
I am not yet aligned with having builders choose to customize the loader by switching any sort of icon/logo prop at this time. This is tightly associated with Design System guidance and best practices for OSD. 

Additional loaders for varying feature cases are being considered.